app/controllers/categories_controller.rb

```diff
@@ -175,4 +183,8 @@ def category_params
     def fetch_category
       @category = Category.find_by(slug: params[:id]) || Category.find_by(id: params[:id].to_i)
     end
+
+    def initialize_staff_action_logger
```

Comment:
https://meta.discourse.org/t/category-security-changes-arent-logged/26820/5

Ahh ok I'll need to support it